variables.tf

```diff
@@ -8,8 +13,97 @@ variable "location" {
   type        = string
 }
 
+variable "log_analytics_workspace_id" {
+  description = "The ID of the Log Analytics workspace to send diagnostics to."
+  type        = string
+}
+
+variable "administrator_login" {
+  description = "The login username of the administrator for this MySQL server."
+  type        = string
+}
+
+variable "administrator_password" {
+  description = "The login password of the administrator for this MySQL server."
+  type        = string
+}
+
+variable "databases" {
+  description = "A list of databases to create on the MySQL Flexible Server."
+  type = list(object({
```

Comment:
Should be `map(object())`, see [best practices](https://equinor.github.io/terraform-baseline/best-practices/resources/#repeatable-resources).